i18n-helpers/src/lib.rs

```diff
@@ -137,6 +140,33 @@ pub fn extract_events<'a>(text: &'a str, state: Option<State<'a>>) -> Vec<(usize
             .enumerate()
             .map(|(idx, line)| (idx + 1, Event::Text(line.into())))
             .collect(),
+        // If we're in a table cell, we put the text in a minimal table, parse the
+        // table, and return the contents of the cell. This matches the behavior of
+        // the parser in this case.
+        Some(state) if state.in_table_cell => {
+            let text = format!("|{}|\n|-|", text);
```

Comment:
I'm fond of the inline formatting directives and use them where possible:

```suggestion
            let text = format!("|{text}|\n|-|");
```